src/log.js

```diff
@@ -391,20 +391,20 @@ export class Log {
       error.message = error.message.replace(USER_ERROR_SENTINEL, '');
     }
   }
-}
-
 
-/**
- * @param {string|!Element} val
- * @return {string}
- */
-function toString(val) {
-  if (val instanceof Element) {
-    return val.tagName.toLowerCase() + (val.id ? '#' + val.id : '');
+  /**
+   * @param {string|!Element} val
+   * @return {string}
+   * @private
+   */
+  toString_(val) {
+    if (val instanceof this.win.Element) {
```

Comment:
Not going to work since multiple windows are involved. Just do `if (val.nodeType) {...}` to test for an element. See `assertElement` in this module.